crates/sail-common/src/spec/expression.rs

```diff
@@ -353,3 +354,28 @@ impl PySparkUdfType {
 pub struct UnresolvedNamedLambdaVariable {
     pub name: ObjectName,
 }
+
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize, Default)]
+#[serde(rename_all = "camelCase")]
+pub struct WildcardOptions {
+    pub ilike_pattern: Option<String>,
+    pub exclude_columns: Option<Vec<Identifier>>,
+    pub except_columns: Option<Vec<Identifier>>,
+    pub replace_columns: Option<Vec<WildcardReplaceColumn>>,
+    pub rename_columns: Option<Vec<IdentifierWithAlias>>,
+}
+
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct WildcardReplaceColumn {
+    pub expression: Box<Expr>,
+    pub column_name: Identifier,
+    pub as_keyword: bool,
```

Comment:
Is this needed? If it is used for differentiating SQL syntax, we probably don't want it in the spec.